lib/rubocop/cop/style/empty_literal.rb

```diff
@@ -67,10 +67,13 @@ def string_literals_config
         end
 
         def first_argument_unparenthesized?(node)
-          return false unless node.parent && node.parent.send_type?
+          parent = node.parent
+          unless parent && %i[send super zsuper].include?(parent.type)
+            return false
+          end
 
-          _receiver, _method_name, *args = *node.parent
-          node.object_id == args.first.object_id && !parentheses?(node.parent)
+          node.object_id == parent.arguments.first.object_id &&
```

Comment:
I think we have been using `#equal?` for identity checks. (Perhaps a cop opportunity. 🙂)